README.md

```diff
@@ -1 +1,33 @@
+# What is the Weather? App
+
+A weather application to demonstrate API calls and Manipulation.
+
+## Install (MAC)
+
+* Install Homebrew ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"```
+* Install the most current version of node.js ```brew install node```
```

Comment:
I like that you included this much detail.
I think its superfluous though.

If there are general things needed for the project sometimes I will include a section under install called requirements.

I like that you included a step to set up your env file.

Installing dependencies for a user of the app will hook into the `package.json` file.
As a user I should not be installing the specific dependencies. I should just be installing everything the project needs.
i.e. `npm install`

Reply:
Done.

Reply:
I would include a line for `npm install` btw.